### PR TITLE
Modify Regular Expression for the support of tag suffix with latest/fast

### PR DESCRIPTION
**Pull request by rpancham**
What this PR does / why we need it:
Automate ODH Release Process for Kserve repos

Which issue(s) this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #[RHOAIENG-2732](https://issues.redhat.com/browse/RHOAIENG-2732)

### CODE REVIEW

File: .github/workflows/create-release.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
               
       - name: Update params.env with new release version
         run: |     
-          sed -i 's|:v[0-9.]*\b|:${{ github.event.inputs.tag_name }}|gm' config/base/params.env
+          sed -i 's|:v[0-9.]*.*|:${{ github.event.inputs.tag_name }}|gm' config/base/params.env
```
[the remark this answers is not in the document]
Comment (rpancham) [reply .github/workflows/create-release.yaml:69]: Yes, it doenot start with v , On the release branches we will be having the tag starting with v but for the first release we will the having it has fast or latest so yesterday it was decided that for the tag we will be prefixing the version. 

@israel-hdez please can you confirm

[the remark this answers is not in the document]
Comment (israel-hdez) [reply .github/workflows/create-release.yaml:69]: My understanding is that this new workflow would only be ran against `release-*` branches, not in `main` branch.
So, as part of our branch cut, we should modify `params.env` file to use `v{something}` tags. Ex: https://github.com/opendatahub-io/odh-model-controller/blob/release-0.11.1/config/base/params.env. Thus, the regex would match for `release-*` branches, which seems to be what we need.

[the remark this answers is not in the document]
Comment (spolti) [reply .github/workflows/create-release.yaml:69]: That sounds good, asked because the PR title confused me.

**Pull request merged**